src/main/kotlin/no/nav/sifinnsynapi/soknad/SøknadService.kt

```diff
@@ -49,20 +48,21 @@ class SøknadService(
             (oppslagsService.hentSøker()
                 ?: throw IllegalStateException("Feilet med å hente søkers aktørId.")).aktørId
 
-        val barnOppslagDTOS: List<BarnOppslagDTO> = oppslagsService.hentBarn()
-        if (barnOppslagDTOS.isEmpty()) {
-            logger.info("Fant ingen barn på søker")
-            return listOf( )
-        }
 
-        val pleietrengendeAktørIder = omsorgService.hentPleietrengendeSøkerHarOmsorgFor(søkersAktørId)
-        if (pleietrengendeAktørIder.isEmpty()) {
+        val pleietrengendeSøkerHarOmsorgFor = omsorgService.hentPleietrengendeSøkerHarOmsorgFor(søkersAktørId)
+        if (pleietrengendeSøkerHarOmsorgFor.isEmpty()) {
             logger.info("Fant ingen pleietrengende søker har omsorgen for.")
             return listOf()
         }
-        logger.info("Fant {} pleietrengende søker har omsorgen for.", pleietrengendeAktørIder.size)
 
-        return pleietrengendeAktørIder
+        val barnOppslagDTOS: List<BarnOppslagDTO> = oppslagsService.systemoppslagBarn(HentBarnForespørsel(identer = pleietrengendeSøkerHarOmsorgFor))
+        if (barnOppslagDTOS.isEmpty()) {
+            logger.info("Fant ingen barn på søker")
+            return listOf( )
```

Comment:
Dette logger du to ganger nå – både i `oppslagsService.systemoppslagBarn` og her. Kanskje du kan droppe denne, for å gjøre loggene litt enklere å parse?

```suggestion
            return barnOppslagDTOS
```